src/components/Drawer.jsx

```diff
@@ -525,12 +525,12 @@ const menuItems = [
   //   subOptions: [],
   //   route: "/dao",
   // },
-  {
-    name: CONTENT_NAMES.ANALYTICS,
-    icon: analyticsIcon,
-    subOptions: [],
-    route: "/analytics",
-  },
+  //{
```

Comment:
Why are we cutting out the Analytics tab?

Reply:
see the convo between Eliana and Ryan on slack internal - I'm happy to shift focus on it to after this weekend but that's up to you guys